spec/features/two_factor_authentication/sign_in_spec.rb

```diff
@@ -127,45 +128,34 @@
         expect(phone_radio_button).to_not be_disabled
       end
 
-      scenario 'updates international code as user types', :js do
-        sign_in_before_2fa
-        fill_in 'Phone', with: '+81 54 354 3643'
-
-        expect(page.find('#user_phone_form_international_code').value).to eq 'JP'
-
-        fill_in 'Phone', with: '5376'
-        select 'Morocco +212', from: 'International code'
-
-        expect(find('#user_phone_form_phone').value).to eq '+212 (537) 6'
-
-        fill_in 'Phone', with: '54354'
-        select 'Japan +81', from: 'International code'
-
-        expect(find('#user_phone_form_phone').value).to include '+81'
-      end
```

Comment:
Why did we remove this spec? Are we removing this feature?

Reply:
The intl-tel dropdown populates the input with the `+XX` prefix when a country is selected from the dropdown and currently does not allow the user to backspace the input to remove the `+XX` prefix. 

When selecting a different country the phone number that was entered gets cleared. 
see screenshot of both switching countries after entering a number and also trying to backspace to remove the prefix.

![untitled](https://user-images.githubusercontent.com/1449852/39927832-949acf70-5501-11e8-89c7-984fcf1c64f1.gif)
![untitled1](https://user-images.githubusercontent.com/1449852/39927840-9904fb80-5501-11e8-8e76-44cd92d07b64.gif)